lib/widgets/drive/drive-scanner.coffee

```diff
@@ -0,0 +1,27 @@
+compare = require('./compare')
+EventEmitter = require('events')
+_ = require('lodash')
+
+module.exports = class DriveScanner extends EventEmitter
```

Comment:
Can you write JSDoc annotations for the class and it's methods? BTW, there references here: http://usejsdoc.org is a good place to check how to annotate certain things.